pkg/controller/s3/bucket/accelerateConfig.go

```diff
@@ -68,7 +69,7 @@ func NewAccelerateConfigurationClient(client s3.BucketClient) *AccelerateConfigu
 // Observe checks if the resource exists and if it matches the local configuration
 func (in *AccelerateConfigurationClient) Observe(ctx context.Context, bucket *v1beta1.Bucket) (ResourceStatus, error) {
 	external, err := in.client.GetBucketAccelerateConfigurationRequest(&awss3.GetBucketAccelerateConfigurationInput{Bucket: aws.String(meta.GetExternalName(bucket))}).Send(ctx)
-	if err != nil {
+	if err != nil || strings.Contains(err.Error(), "MethodNotAllowed") || strings.Contains(err.Error(), "UnsupportedArgument") {
 		return NeedsUpdate, errors.Wrap(err, accelGetFailed)
```

Comment:
I believe if the error is `MethodNotAllowed`, the returned status should be `Updated` since there is no operation to be done here.

Reply:
Good point.

Reply:
Ah I messed up the logic, I did this late last night. Should be and does not contain.